src/main/java/mate.academy/service/ProductService.java

```diff
@@ -1,8 +1,10 @@
 package mate.academy.service;
 
 import java.util.List;
+import mate.academy.lib.Component;
 import mate.academy.model.Product;
 
+@Component
 public interface ProductService {
```

Comment:
Convention is to use Component on the classes that are used for instance creation (usually these are interface implementations)